thrift0.13/src/main/java/com/linecorp/armeria/common/thrift/ThriftSerializationFormats.java

```diff
@@ -15,16 +15,23 @@
  */
 package com.linecorp.armeria.common.thrift;
 
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.collect.ImmutableMap.toImmutableMap;
 import static java.util.Objects.requireNonNull;
 
+import java.util.List;
+import java.util.Map;
+import java.util.ServiceLoader;
 import java.util.Set;
 
-import com.google.common.collect.ImmutableSet;
+import org.apache.thrift.protocol.TProtocolFactory;
+
+import com.google.common.collect.ImmutableList;
 
 import com.linecorp.armeria.common.SerializationFormat;
 
 /**
- * Thrift-related {@link SerializationFormat} instances.
+ * Registered Thrift-related {@link SerializationFormat} instances.
```

Comment:
How about this:

```
Provides Thrift-related {@link SerializationFormat} instances and their {@link TProtocolFactory}s.
```

.. because whether this class registers anything is implementation detail.